src/index.js

```diff
@@ -4,9 +4,14 @@ import TodoList from '../modules/workClass.js';
 import addTodo from '../modules/addTodo.js';
 import deleteTodos from '../modules/deleteTodos.js';
 import editTodo from '../modules/editTodos.js';
+import interactivelist from '../modules/interactivelist.js';
+import { clearcompletedtodos, refreshpage } from '../modules/clearcompletedtodos.js';
 
 const todoclass = new TodoList();
 todoclass.workList();
 addTodo();
 deleteTodos();
 editTodo();
+interactivelist();
+clearcompletedtodos();
+refreshpage();
```

Comment:
- [ ] You did a good job working on this milestone! 💯 However, please check out this:

![image](https://github.com/rabayarabu/To-Do-List/assets/77069011/b99e84de-c04f-43ac-970e-ee9fd53f4713)

Currently, your to-dos app isn't working as expected, when I try to check a To-do as completed, it doesn't change the style or update the local storage. In order for it to work I have to refresh the page, which isn't a good user experience. This happens because the event listeners only get added when the function `interactivelist` is called, and you're calling it here at page load. This causes that only previous existing to-dos have event-listeners and the ones you create new doesn't. Please consider fixing this behavior! 🤓 🔧 

The same happens with your edit function, I'm unable to edit a to-do unless I refresh the page first.